engine/src/main/java/org/terasology/engine/input/lwjgl/LwjglControllerDevice.java

```diff
@@ -92,6 +92,7 @@ private void updateGamepadMappings() {
             if (!GLFW.glfwUpdateGamepadMappings(gamecontrolleDB)) {
                 logger.error("Cannot update GLFW's gamepad mapping, gamepads may not work correctly");
             }
+            inputStream.close();
```

Comment:
Use [try with resources](https://www.baeldung.com/java-try-with-resources) instead of manually closing. This hopefully applies to most of the occurrences of this warning.

Reply:
done